### PR TITLE
Added PcapConvert, PcapExtractStreams and PcapFileExtractStreams.

### PR DESCRIPTION
**Pull request by spearmin10**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
[fixes: link to the issue](https://github.com/demisto/content/pull/18650#issuecomment-1123867462)

## Description
Added PcapConvert, PcapExtractStreams and PcapFileExtractStreams.

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by JasBeilin — COMMENTED** [review]
Thank you for your contribution!
Please remove the pipfile as we are looking at the docker image and do not use the pipfile.

File: Packs/PcapAnalysis/Scripts/PcapExtractStreams/PcapExtractStreams_test.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import demistomock as demisto
+import json
+
+
+def test_main(mocker):
+    from PcapExtractStreams import main
```
Comment (JasBeilin) [inline Packs/PcapAnalysis/Scripts/PcapExtractStreams/PcapExtractStreams_test.py:6]: Please add given-when-then doc string

File: Packs/PcapAnalysis/Scripts/PcapExtractStreams/PcapExtractStreams_test.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+            'error_action': t.get('error_action'),
+            'server_ports': t.get('server_ports'),
+        })
+        mocker.patch.object(demisto, 'results')
+        main()
+        assert demisto.results.call_count == 1
+        results = demisto.results.call_args[0][0]
+        if json.dumps(results) != json.dumps(t['result']):
+            print(json.dumps(t['result']))
+            print(json.dumps(results))
```
Comment (JasBeilin) [inline Packs/PcapAnalysis/Scripts/PcapExtractStreams/PcapExtractStreams_test.py:30]: Please remove printing

**Review by JasBeilin — CHANGES_REQUESTED** [review]

File: Packs/PcapAnalysis/Scripts/PcapFileExtractStreams/PcapFileExtractStreams.py [hunk 3]
```diff
@@ -0,0 +1,364 @@
...
+        """
+        # Create custom parameters
+        custom_parameters = None
+        if rsa_key_file_path:
+            custom_parameters = {'-o': f'uat:rsa_keys:"{rsa_key_file_path}",""'}
+
+        # Parse the pcap
+        logging.getLogger('asyncio').setLevel(logging.ERROR)
+        with open(os.devnull, 'w') as devnull:
+            sys.stderr = devnull
```
[the remark this answers is not in the document]
Comment (spearmin10) [reply Packs/PcapAnalysis/Scripts/PcapFileExtractStreams/PcapFileExtractStreams.py:272]: pyshark sometimes crashes and writes error messages when parsing incorrect pcap.  In that case, this kind of results returns even when excepting and dismissing the errors. 

image

The code prevents it and make expected results (null) like this:
image

**Comment by spearmin10** [comment]
@JasBeilin Updated test scripts following to your comments. Please review again.

**Review by JasBeilin — APPROVED** [review]

**Comment by ShirleyDenkberg** [comment]
@JasBeilin Doc review completed.

**Pull request merged**